pkg/ingester/ingester_v2.go

```diff
@@ -458,6 +460,7 @@ func (i *Ingester) createTSDB(userID string) (*tsdb.DB, error) {
 		}()
 	}
 
+	i.TSDBState.tsdbMetrics.setRegistryForUser(userID, tsdbPromReg)
```

Comment:
Could you add the two new metrics to `ingester_v2_test.go` wherever the `cortex_ingester_memory_series` is checked, please?